wordpress-mobile/AztecEditor-iOS
# Add test to check nested list output.

Fixes #1164 

At the moment this only adds a unit test to verify the bug

To test:
 - Open the demo empty app
 - Copy paste the following HTML:
```
<ul>
  <li>List 1</li>
  <li>List 2
           <ul>
              <li>List 3</li>
              <li>List 4</li>
            </ul>
  </li>
</ul>
```
 - Switch to visual and back to HTML and see if the output is correct.
